src/main/java/org/apache/commons/io/FilenameUtils.java

```diff
@@ -1510,6 +1510,9 @@ public static boolean wildcardMatch(final String fileName, final String wildcard
      * @return true if the given name is a valid host name
      */
     private static boolean isValidHostName(final String name) {
+        if(Objects.isNull(name)) {
```

Comment:
Fix formatting and no need to call an API meant for use in lambdas when `name == null` would do.